### ISSUE TITLE
why using the S3dis dataset to train but the class 7、８、9 is always zero?

### ISSUE THREAD

**Issue by laiming997** [aRI0U/RandLA-Net-pytorch#8]
I just download the code , and using the Stanford3dDataset_v1.2_Aligned_Version as dataset, but when I have trained the model, the param is epoch=200, batch_size=3, the overall accuracy is almost 0.65+, and many class is 0, only class0 and class1 、class2 look normal,  the ActiveLearningSampler can make sure the model can see all of the data?

**Comment by Thibaud-Ardoin** [comment]
Hey, thanks for your interest in our implementation !

The general problem of this DataSet is the distribution of the classes. The 'Wall' classe is represented up to 70 times more than other classes. Therefore, the small classes are washed out and get 0% accuracy.
The ActiveLearningSampler is used to get a faire distribution of the point clouds, but not the classes.

This issue is probably responsible for the gap in performances with the original paper. 
You are welcome to open a Pull Request if you have any solution on your side

Cheers,

Thibaud

**Comment by JerryIndus** [comment]
> I just download the code , and using the Stanford3dDataset_v1.2_Aligned_Version as dataset, but when I have trained the model, the param is epoch=200, batch_size=3, the overall accuracy is almost 0.65+, and many class is 0, only class0 and class1 、class2 look normal, the ActiveLearningSampler can make sure the model can see all of the data?

Hi, have you had any solution to this question? I also meet the same problem but have no idea...

**Comment by wangzihao77** [comment]
> > 我只是下载了代码，并使用Stanford3dDataset_v1.2_Aligned_Version作为数据集，但当训练模型时，参数是epoch=200，batch_size=3，整体精度几乎是0.65+，许多类是0，只有class0和class1、class2看起来正常，ActiveLearningSampler可以确保模型可以看到所有数据？
> 
> 嗨，你对这个问题有什么解决方案吗？我也遇到了同样的问题，但不知道......

**Comment by wangzihao77** [comment]
> > 我只是下载了代码，并使用Stanford3dDataset_v1.2_Aligned_Version作为数据集，但当训练模型时，参数是epoch=200，batch_size=3，整体精度几乎是0.65+，许多类是0，只有class0和class1、class2看起来正常，ActiveLearningSampler可以确保模型可以看到所有数据？
> 
> 嗨，你对这个问题有什么解决方案吗？我也遇到了同样的问题，但不知道......

hi,me too,can you solve it? we can talk about it.
